templates/rapid-openai/pages/api/rapid.ts

```diff
@@ -0,0 +1,37 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import axios from "axios";
+
+// Prompt for API key
```

Comment:
Does this prompt show up when a call to the serverless function is made?

Reply:
@ahmadbilaldev ^

Reply:
Yes, only when the call is made.